### DISCUSSION TITLE
Console to report (spreadsheet)

### DISCUSSION THREAD

**Discussion by donert** [togrupe/dlive-midi-tools#35] [Ideas]
I see in the planned features "Print current channels" - I wonder if this might be similar to a "Console to Spreadsheet" mode. My use case is setting up a new console , I would like to be able to export the show and then share the settings for review with other engineers. This is before I even have the hardware. Such a feature could also be used for round-tripping; grab the current config from the hardware, edit it in excel, and then reapply to the console.

**Comment by togrupe** [comment]
Hi @donert ,

Thanks for your feature suggestion.

The feature "Console to Spreadsheet" is planned, in Wiki I named it "Read back the dlive console parameters" & "Export spreadsheets" which is basically the same idea. 

With "print current channels" I thought of a short version of the spreadsheet, for patching on stage. Let me check if a can make it for the next release.

Are you using the tool already? Happy with it?

**Reply by donert** [reply]
Great. I am not using it yet. We just got approval to purchase an s5000 and dm64. Will get my hands on it this summer, hopefully. I have been doing a lot of study on the system and tools that can help run it.

**Reply by togrupe** [reply]
cool, great choice. If you want, you can already run first tests together with director (offline editor), I also drive my development this way. Final tests are of course with real hardware.

**Comment by crazzell** [comment]
I also have a use case where I primarily want to audit configurations that are currently stored in the mixer. The basic channel list currently available in PDF or CSV form is already useful, but not sufficiently complete for things like routing tables. I think the completion of a roundtrip option to allow a mixer's existing configuration to be edited in spreadsheet land would be highly useful. Admittedly, proprietary off-line editors already do some of what I'm asking but I'm missing the ability to save mixer configuration data in human-readable form for group discussion, validation and archiving.
